### ISSUE TITLE
A11Y Lab - Contrast (Enhanced)

### ISSUE THREAD

**Issue by glenn-sorrentino** [scidsg/hushline#160]
1.4.6.Contrast (Enhanced): The visual presentation of text and images of
text has a contrast ratio of at least 7:1, except for the following:
• Large Text: Large-scale text and images of large-scale text have a contrast ratio
of at least 4.5:1;
• Incidental: Text or images of text that are part of an inactive user interface
component, that are pure decoration, that are not visible to anyone, or that are
part of a picture that contains significant other visual content, have no contrast
requirement.
• Logotypes: Text that is part of a logo or brand name has no minimum contrast
requirement.
See 1

**Comment by glenn-sorrentino** [comment]
file:///Users/glennsorrentino/Downloads/WCAG%202.1%20AAA_HushLine%20website.pdf

**Comment by glenn-sorrentino** [comment]
👍 Fixed all contrast issues
![Screenshot 2023-10-19 at 12 31 59 PM](https://github.com/scidsg/hushline/assets/28545431/d9870bdb-83af-4e0e-a20e-e59b201b7e37)
